configs/pie-components-config/vite.config.js

```diff
@@ -10,7 +10,7 @@ const bundledJetDeps = [
 ];
 
 // https://vitejs.dev/config/
-const sharedConfig = ({ build = {}, plugins = [], ...rest }) => defineConfig({
+export const sharedConfig = ({ build = {}, plugins = [], ...rest }) => defineConfig({
```

Comment:
If you're changing this from a default to a named export you'll need to update all the references (and the docs)